src/celeritas/ext/detail/GeantGeometryImporter.hh

```diff
@@ -0,0 +1,145 @@
+//----------------------------------*-C++-*----------------------------------//
+// SPDX-FileCopyrightText: 2020 CERN
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
At this point (or at least after we wrap up this MR) I think we'll have basically rewritten these entire files... not sure how we should attribute a work that's heavily modified from an original...

Reply:
I've modified these appropriately.